### DISCUSSION TITLE
How to add custom svg for arrow?

### DISCUSSION THREAD

**Discussion by Parthvsquare** [gilbarbara/react-floater#104] [Q&A] [answered]
I tried to add content, but it was not showing up? is there any other way to add custom svg for arrow?
<img width="784" alt="image" src="https://github.com/gilbarbara/react-floater/assets/26404957/e171e42a-7c3f-4f0f-8eb8-a5c8d531e513">

**Comment by gilbarbara** [comment, marked as answer]
Hey @Parthvsquare 
You can't change currently. You can check the valid properties for the arrow [here](https://github.com/gilbarbara/react-floater/blob/main/src/modules/styles.ts).

**Comment by ghost** [comment]
is the custom arrow available yet ?
